tests/func/clean/all.yml

```diff
@@ -1,41 +1,47 @@
-- include_tasks:
-    file: router.yml
+---
+- name: Include router tasks
+  include_tasks: router.yml
+  args:
```

Comment:
Based on 4th example from: https://docs.ansible.com/ansible/latest/modules/include_tasks_module.html

Reply:
Ack this is good.